src/platform_impl/linux/x11/util/cursor.rs

```diff
@@ -1,13 +1,20 @@
 use crate::window::CursorIcon;
+use std::mem;
 
 use super::*;
 
 impl XConnection {
     pub fn set_cursor_icon(&self, window: ffi::Window, cursor: Option<CursorIcon>) {
+        // ignore custom cursors for now
+        let cursor = match cursor {
+            Some(CursorIcon::Custom(_)) => Some(CursorIcon::default()),
```

Comment:
What is the point of filtering out cursor here, if you can safely do that in `get_cursor`? You have already and arm for that in a match, just load `left_ptr` as you do on `web` and `Wayland` now?

Reply:
Good point. That's been corrected.